File: spoj/FCTRL.cpp
#include<iostream>
using namespace std;
typedef long long ll;

ll getZeroes(ll n){
	ll cnt = 0;
	while(n > 0){
		n = n / 5;
		cnt += n;
	}
	return cnt;
}

int main(){
	int N;
	cin >> N;
	while(N-- > 0){
		ll n;
		cin>>n;
		cout<<getZeroes(n)<<endl;
	}
}

File: spoj/FCTRL.go
package main

import (
	"fmt"
)

func getZeroes(x int64) int64 {
	var cnt int64
	for x > 0 {
		x = x / 5
		cnt += x
	}
	return cnt
}

func main() {
	var N int
	fmt.Scanf("%d", &N)
	for ; N > 0; N-- {
		var x int64
		fmt.Scanf("%d", &x)
		fmt.Println(getZeroes(x))
	}
}
